### PR TITLE
Create 2019-01-11-Episode-11-Dask.md

### PR DESCRIPTION
**Pull request by bsodenkamp**

### CODE REVIEW

File: _posts/2019-01-11-Episode-11-Dask.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### Demo
+
+[Demo at timestamp 11:32](https://www.youtube.com/watch?v=u-HAzanaRVI)
+[Github Repository](https://github.com/dask/dask)
+[examples that are publicly runnable on binder up here](http://examples.dask.org/)
+[Dataframe](https://mybinder.org/v2/gh/dask/dask-examples/master?urlpath=lab/tree/dataframe.ipynb)
+[Delayed](https://mybinder.org/v2/gh/dask/dask-examples/master?urlpath=lab/tree/delayed.ipynb)
+[Surface Sea Ice](http://binder.pangeo.io/v2/gh/pangeo-data/pangeo-example-notebooks/master)
+
+### Rolling Goals
```
Comment (scopatz) [inline _posts/2019-01-11-Episode-11-Dask.md]: Change this to "Wishlist" please

File: _posts/2019-01-11-Episode-11-Dask.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+---
+
+**Q.** The Dask Stream seems a cool visualization tool for tracking computations. Say, if one would have a Python script doing computations (not using dask), would it be possible to visualize the computations using the Dask Stream feature? Say, by wrapping the script main function with Dask or smth?
+
+**A.** Ya, so, the task stream shows computations that Dask knows about.  From a user perspective if you’re trying to write some custom thing you could use Dask-delayed, and we also have a futures interface if you are worried about futures.  Then you would be delegating certain functions to show up in the past stream.  As those are computed, they would show up in the past stream.
```
Comment (scopatz) [inline _posts/2019-01-11-Episode-11-Dask.md]: Remove the "Ya, so"

File: _posts/2019-01-11-Episode-11-Dask.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+---
+
+**Q.** Does anyone have work/plans/interest in automatic differentiation of calculations done with Dask?
+
+**A.** I don’t think anyone on this webinar has plans for that, but if you are interested in it then it would certainly be worth looking into.  We would be curious to see that as well and maybe help out with it.  One interesting thing on that is there are a couple libraries that do automatic differentiation based on NumPy, so following the NumPy API.  Since Dask follows the NumPy API, if you could make it separate from NumPy then it would likely just work with Dask array. 
+
+### Footnotes & Links
+
+* Roadmap [here](https://github.com/dask/dask-ml/issues?q=is%3Aopen+is%3Aissue+label%3ARoadmap)
```
Comment (scopatz) [inline _posts/2019-01-11-Episode-11-Dask.md]: Wishlist instead of a roadmap

**Comment by bsodenkamp** [comment]
Made those edits

**Comment by scopatz** [comment]
Great! Thanks!

**Pull request merged**